api/sys_monitor.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"bufio"
+	"context"
+)
+
+func (c *Sys) Monitor(logLevel string, stopCh chan struct{}) (chan string, error) {
```

Comment:
Instead of a stopCh, why not take a context?

Reply:
An excellent question!  I have no idea 😁 I don't know enough about contexts to answer your question in an intelligent manner.  Mostly I was following patterns from how nomad and consul implemented their monitor commands, and they both used a stopCh, so that's how this ended up here.  I'd be interested in learning more about contexts though.